src/asm3/media.py

```diff
@@ -746,6 +847,17 @@ def get_signature_link(dbo: Database, mid: int, post: PostedData) -> str:
     url = "%s?account=%s&method=sign_document&email=%s&formid=%d&token=%s" % (SERVICE_URL, dbo.name(), asm3.utils.strip_email_address(post["to"]).replace("@", "%40"), mid, token)
     return url
 
+def get_video_thumbnail(dbo: Database, dbfsid: int) -> bytes:
+    inputfile = tempfile.NamedTemporaryFile(suffix=".mp4", delete=False)
+    vdata = asm3.dbfs.get_string_id(dbo, dbfsid)
+    inputfile.write(vdata)
+    inputfile.close()
+    outputfile = tempfile.NamedTemporaryFile(suffix=".jpg", delete=False)
+    asm3.utils.cmd(VIDEO_THUMBNAIL_CMD % { "output": outputfile.name, "input": inputfile.name}, False)
```

Comment:
The return code and/or output should be checked here, if the return code is non-zero then we know something went wrong and can raise one of our HTTP flavoured exceptions, eg: asm3.utils.ASMError